### ISSUE TITLE
Input doesn't get cleared

### ISSUE THREAD

**Issue by JackyChiu** [Hacker-Chatroom/Hacker-Chatroom-Client#12]
After an input is entered it is still left in the line:
```
>hello # This should be cleared
> # and this
15:30 : Jacky
- hello
>
```
<img width="107" alt="screen shot 2016-11-07 at 7 21 57 pm" src="https://cloud.githubusercontent.com/assets/11095732/20081769/9415d98e-a51f-11e6-91da-e1807d56b4d4.png">

**Comment by BenEmdon** [comment]
@JackyChiu is this fixed by your latest PR or is it already in the build?

**Comment by JackyChiu** [comment]
@BenEmdon The portion that could be fixed was fixed by PR #13. However the part that isn't fixable is the echo of the input ex:
```
>hello # This should be cleared
15:30 : Jacky
- hello
>
``` 
This is something that's built into terminals. We would need another technology to fix this.
